spacy/vectors.pyx

```diff
@@ -339,7 +339,7 @@ cdef class Vectors:
             return self.key2row.get(key, -1)
```

Comment:
I realize it's only one key, but doesn't it make sense to make the same change here, too?

Reply:
Done.